corehq/apps/hqwebapp/static/hqwebapp/js/base_main.js

```diff
@@ -11,6 +11,7 @@ hqDefine("hqwebapp/js/base_main", [
     'analytix/js/hubspot',
     'analytix/js/kissmetrix',
     'hqwebapp/js/mobile_experience_warning',
+    'hqwebapp/js/modernizr',
```

Comment:
Why is this being added to base_main.js? This is only for scripts that are ~always included in base.html; I thought this PR only affected the signup flow.

Reply:
Nevermind, I read further down and see what you're doing here.